cmd/podman/ps.go

```diff
@@ -549,24 +632,57 @@ func getJSONOutput(containers []*libpod.Container, opts batchcontainer.PsOptions
 
 		psOutput = append(psOutput, params)
 	}
+	return sortPsOutput(opts.Sort, psOutput), nil
+}
+
+// getJSONOutput returns the container info in its raw form
+func getJSONOutput(psParams psSorted, opts batchcontainer.PsOptions) ([]psJSONParams, error) {
+	var (
+		psOutput []psJSONParams
+	)
+	for _, psParam := range psParams {
+		params := psJSONParams{
```

Comment:
Do we need a separate psParams if psJSONParams is basically identical? The only real difference is status.String() and getMountsArray() - can we massage those to use psJSONParams for everything instead of psParams?